helpers/git.js

```diff
@@ -32,12 +32,19 @@ var fs = require('fs');
  * @throws {Error} If cannot get last git commit
  */
 function getLastCommit() {
-	var spawn = childProcess.spawnSync('git', ['rev-parse', 'HEAD']);
-	var err = spawn.stderr.toString().trim();
+	let spawn;
 
-	// If there is git tool available and current directory is a git owned directory
-	if (!err) {
-		return spawn.stdout.toString().trim();
+	try {
+		spawn = childProcess.spawnSync('git', ['rev-parse', 'HEAD']);
+		// If there is git tool available and current directory is a git owned directory
+		if (!spawn.stderr) {
```

Comment:
As per documentation `stderr` will always be returned as string or Buffer. 
https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options . In case of empty string this logic may work, but in case of empty buffer it will fail. So better o use the same logic which was used earlier `!pawn.stderr.toString().trim()`